### PR TITLE
SUS-2257 → remove obsolete references to post_user_ip

### PR DESCRIPTION
**Pull request by ghost**
https://wikia-inc.atlassian.net/browse/SUS-2257

This is a cleanup after https://github.com/Wikia/app/pull/13416/files

### CODE REVIEW

**Comment by WikiaTech** [comment]
SonarQube analysis reported 5 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [WallHistory.class.php#L28](https://github.com/Wikia/app/blob/c8bc78ecdb27cc04f4ddc2094510ad81bd8e12bb/extensions/wikia/Wall/WallHistory.class.php#L28): Add a "case default" clause to this "switch" statement. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS131)
1. ![MAJOR][MAJOR] [WallHistory.class.php#L152](https://github.com/Wikia/app/blob/c8bc78ecdb27cc04f4ddc2094510ad81bd8e12bb/extensions/wikia/Wall/WallHistory.class.php#L152): This function has 11 parameters, which is greater than the 7 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS107)
1. ![MAJOR][MAJOR] [WallHistory.class.php#L438](https://github.com/Wikia/app/blob/c8bc78ecdb27cc04f4ddc2094510ad81bd8e12bb/extensions/wikia/Wall/WallHistory.class.php#L438): Replace this legacy wfMsg* function call with one of the new i18n functions. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3ALegacyWfMsgUsage)
1. ![MINOR][MINOR] [WallHistory.class.php#L369](https://github.com/Wikia/app/blob/c8bc78ecdb27cc04f4ddc2094510ad81bd8e12bb/extensions/wikia/Wall/WallHistory.class.php#L369): Immediately return this expression instead of assigning it to the temporary variable "$res". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1488)
1. ![INFO][INFO] [WallHistory.class.php#L184](https://github.com/Wikia/app/blob/c8bc78ecdb27cc04f4ddc2094510ad81bd8e12bb/extensions/wikia/Wall/WallHistory.class.php#L184): Move this trailing comment on the previous empty line. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)

[project-id]: wikia:app
[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Review by macbre — APPROVED** [review]

File: extensions/wikia/Wall/sql/wall_history_local.sql [hunk 1]
```diff
@@ -1,7 +1,6 @@
 CREATE TABLE wall_history (
 	parent_page_id INT(8) UNSIGNED,
 	post_user_id INT(10) UNSIGNED,
-	post_user_ip INT UNSIGNED,
```
Comment (macbre) [inline extensions/wikia/Wall/sql/wall_history_local.sql:4]: Let's add a drop of `post_user_ip` column to update.php.

Comment (ghost) [reply extensions/wikia/Wall/sql/wall_history_local.sql:4]: I've updated `WikiaUpdater` to reflect the entire migration, not just the final drop.

**Comment by ghost** [comment]
```
michal@dev-michal:/usr/wikia/source/wiki$ SERVER_ID=831 php maintenance/eval.php
> WikiaUpdater::do_wall_history_ipv6_update(DatabaseUpdater::newForDB(wfGetDB(DB_MASTER)));
starting WikiaUpdater::do_wall_history_ipv6_update...
post_user_ip_bin already exists in wall_history but it is OK...
migrating data from post_user_ip to post_user_ip_bin...
dropping post_user_ip in wall_history...
done.

> WikiaUpdater::do_wall_history_ipv6_update(DatabaseUpdater::newForDB(wfGetDB(DB_MASTER)));
starting WikiaUpdater::do_wall_history_ipv6_update...
wall_history has already been migrated, skipping the update.
```

**Review by macbre — APPROVED** [review]

**Comment by macbre** [comment]
Backported to production branch - 9de0479

**Pull request merged**